exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusCollectorTest.java

```diff
@@ -55,23 +55,27 @@ void registerToDefault() throws IOException {
 
   private static ImmutableList<MetricData> generateTestData() {
     return ImmutableList.of(
-        MetricData.create(
+        MetricData.createDoubleSum(
```

Comment:
previously this was a long sum; seems like there would be value in testing both long and double here, rather than only double.

Reply:
Sorry :) good catch.